tests/unit/common/test_validator.py

```diff
@@ -15,7 +15,7 @@
         ("1h", True),
         ("1d", True),
         ("1w", True),
-        ("1y", True),
+        ("1y", False),  # Deprecated unit
```

Comment:
BREAKING [yY] is not a valid unit. e.g. pd.Timedelta("1y") ==> Error

Reply:
@YS-L Pandas does provide future warning for y and Y units in time delta. If we want to maintain the functionality we will need to parse this ourselves

Reply:
This is only used to store the datetime duration. Internally, all these values will be converted to seconds & stored into the persistent. For this case, it should be safe to remove it.